packages/web/src/fixtures/github/cache-path.ts

```diff
@@ -1,4 +1,6 @@
 export const SWRCachePath = {
   getMarketInformation: (marketAddress: string) =>
-    `https://raw.githubusercontent.com/dev-protocol/assets/main/market/${marketAddress}/info.json`
+    `https://raw.githubusercontent.com/dev-protocol/assets/main/market/${marketAddress}/info.json`,
+  getPolicyInformation: (policyAddress: string) =>
+    `https://raw.githubusercontent.com/dev-protocol/assets/main/market/${policyAddress}/info.json`
```

Comment:
I replaced target branch from master to main.

Reply:
Thank you!